client/src/pages/SplashScreen.js

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components/macro'
+import { useHistory } from 'react-router-dom'
+
+export default function SplashScreen({ onBoot, onComplete }) {
+  const history = useHistory()
+
+  useEffect(() => onBoot(), [])
+
+  useEffect(() => {
+    setTimeout(() => {
```

Comment:
hierzu hatte Jerry noch was in Caros Review gesagt.. da ging es darum, dass wenn der user trotzdem was klickt oder so, dass es dann nicht abstürzt - aber ich weiß nicht, ob das in deinem Fall zutrifft? Es gibt ja im Prinzip nichts zu klicken zu dem Zeitpunkt oder?

Reply:
stürzt nicht ab